bip-0009/assignments.mediawiki

```diff
@@ -0,0 +1,37 @@
+==Deployments==
+
+List of proposed deployments.
+
+State can be defined, active, failed. Dates are in UTC.
+
+{| class="wikitable"
```

Comment:
really? seems to render just fine https://github.com/btcdrak/bips/blob/bip9assign/bip-0009/assignments.mediawiki what am I missing?